packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -854,7 +854,8 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
       const minWidth = column.minWidth || MIN_COLUMN_WIDTH;
       const overflowWidth = totalWidth - availableWidth;
 
-      if (column.calculatedWidth! - minWidth >= overflowWidth || !(column.isCollapsable || column.isCollapsible)) {
+      // tslint:disable-next-line:deprecation
+      if (column.calculatedWidth! - minWidth >= overflowWidth || !(column.isCollapsible || column.isCollapsable)) {
```

Comment:
I don't think the order will make a difference here since the condition will succeed if any operand is true

Reply:
Yeah, it doesn't really matter, I moved them just to have a consistent style where the non-deprecated prop appears first. Just a stylistic choice since, like you said, the order doesn't make any difference in execution.